### DISCUSSION TITLE
Entering the skin editor after selecting a beatmap causes unexpected behaviour

### DISCUSSION THREAD

**Discussion by SliceCraft** [ppy/osu#22102] [Q&A]
### Type

Game behaviour

### Bug description

When you open the skin editor after a beatmap has been selected osu will treat the gameplay as if you're still playing it even though you're in the skin editor.
I would expect the beatmap to be played by autoplay just like it would do when you open the skin editor normally or I would expect the skin editor to not open when you try to open it during gameplay.

### Screenshots or videos

https://user-images.githubusercontent.com/51338223/211360303-96eaba01-f9d7-4c19-811f-4b8acacb7069.mp4

### Version

2022.1228.0-lazer

### Logs

[runtime.log](https://github.com/ppy/osu/files/10375149/runtime.log)
[updater.log](https://github.com/ppy/osu/files/10375150/updater.log)
[database.log](https://github.com/ppy/osu/files/10375151/database.log)
[network.log](https://github.com/ppy/osu/files/10375153/network.log)
[performance.log](https://github.com/ppy/osu/files/10375154/performance.log)

**Comment by peppy** [comment]
This is intended, and allows making changes in breaks / while paused. I'm open for better suggestions, but having auto take over does not seem like a good direction as it would break this use case.

**Reply by bdach** [reply]
I guess the direct path is that the binding could be blocked while not paused / during a break. I believe we already have similar logic for other overlays, which also includes auto-dismiss on break end, so this is potentially very easy (not that I checked).

**Reply by peppy** [reply]
I guess that could work. Although it's a hard shortcut to accidentally trigger.
